components/automate-chef-io/content/docs/configuration.md

```diff
@@ -379,6 +379,21 @@ Uncomment and change settings as needed, and then run `chef-automate config patc
 # key = "-----BEGIN RSA PRIVATE KEY-----\n<your load balancer private key>\n-----END RSA PRIVATE KEY-----\n"
 ```
 
+#### Configure maximum number of inflight data collector requests
```

Comment:
Would it be useful to provide some context around when it would be helpful to change this from the default value?

Reply:
```suggestion
#### Configure Inflight Data Collector Request Maximum
```

Reply:
Let's get the documentation in & follow up on the context @mjingle ?